Source/WebCore/platform/graphics/FontCache.cpp

```diff
@@ -223,9 +223,9 @@ FontPlatformData* FontCache::cachedFontPlatformData(const FontDescription& fontD
     // Leading "@" in the font name enables Windows vertical flow flag for the font.
     // Because we do vertical flow by ourselves, we don't want to use the Windows feature.
     // IE disregards "@" regardless of the orientation, so we follow the behavior.
-    const String& familyName = passedFamilyName.substring(passedFamilyName[0] == '@' ? 1 : 0);
+    const AtomString& familyName = StringView(passedFamilyName).substring(passedFamilyName[0] == '@' ? 1 : 0).toAtomString();
```

Comment:
Seems like this should be `auto` (`AtomString`) rather than `const AtomString&`. Less dependent on the subtleties of object lifetime of objects used to initialize constant references.

Reply:
Yes, fixing.